gcc/testsuite/rust/execute/torture/builtin_macros1.rs

```diff
@@ -0,0 +1,18 @@
+// { dg-output "rust/execute/torture/builtin_macros1.rs" }
+macro_rules! file {
```

Comment:
does this need to be defined if its a builtin macro?

Reply:
Ah that's interesting we could use that attribute as a mechanism to mark the NodeId over to a built-in definition. This will make more sense when we start working on imports and visibility though.